### ISSUE TITLE
Can't access errors for nested properties

### ISSUE THREAD

**Issue by bcornett** [turner-industries/formik-semantic-ui#10]
Using an object that has nested properties:
```
{
    name: '',
    physicalAddress: {
        addressLine1: '',
        addressLine2: '',
        city: '',
        state: '',
        zip: '',
    }
}
```
And an component like:
```
<Input label="Address 1" name="physicalAddress.addressLine1" />
```
My errors won't show up for any `physicalAddress` item. I believe this is because of the way you're trying to access the errors in the `helper.js` on line 7 - `form.errors[name]`.

These may be helpful:
https://github.com/jaredpalmer/formik/issues/413
http://blog.nicohaemhouts.com/2015/08/03/accessing-nested-javascript-objects-with-string-key/

**Comment by bcornett** [comment]
Using something like I mentioned in the 2nd link worked: `getNested(form.errors, name)`

**Comment by justinobney** [comment]
closed in #13
